client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -42,7 +46,7 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
 
     const handleIsolation = (value: boolean) => {
         const contactWithIsolationRequirement = {...interactedContact, doesNeedIsolation: value};
-        const validation = validateContact(contactWithIsolationRequirement);
+        const validation = validateContact(contactWithIsolationRequirement, true);
```

Comment:
Having a boolean parameter is considered a bad practice because it's hard to understand what its suppose to do. I recommend composing two `validateContact` functions so each of them handles the `true` and `false` scenario.